### ISSUE TITLE
Add annotations to distinguish the evaluation results of the training/validation set

### ISSUE THREAD

**Issue by Echo9573** [sql-machine-learning/sqlflow#1453]
Added annotations to distinguish the evaluation results of the training set and the validation set. Just like Evaluation result for training set/ Evaluation result for validation set.
